scene/resources/mesh.h

```diff
@@ -159,11 +159,43 @@ class Mesh : public Resource {
 	Size2i get_lightmap_size_hint() const;
 	void clear_cache() const;
 
-	typedef Vector<Vector<Face3>> (*ConvexDecompositionFunc)(const Vector<Face3> &p_faces, int p_max_convex_hulls);
+	struct ConvexDecompositionSettings {
+		enum Mode : int {
+			CONVEX_DECOMPOSITION_MODE_VOXEL = 0,
+			CONVEX_DECOMPOSITION_MODE_TETRAHEDRON
+		};
+
+		/// Maximum concavity. [Range: 0.0 -> 1.0]
+		real_t max_concavity = 1.0;
+		/// Controls the bias toward clipping along symmetry planes. [Range: 0.0 -> 1.0]
+		real_t symmetry_planes_clipping_bias = 0.05;
+		/// Controls the bias toward clipping along revolution axes. [Range: 0.0 -> 1.0]
+		real_t revolution_axes_clipping_bias = 0.05;
+		real_t min_volume_per_ch = 0.0001;
+		/// Maximum number of voxels generated during the voxelization stage.
+		uint32_t resolution = 10'000;
```

Comment:
Yes, the idea was to keep resolution lower so that the generated shape is much less complex.

Reply:
Does it reduce the number of vertices in some cases, or just the accuracy to speed up the process? In the examples I've tested with max vertices 32, it generates convex hulls with 32 vertices in either case.

If it helps simplifying further in some cases, it might be a good example of a parameter that can be in advanced settings, but I would still keep the more accurate value by default.